### PR TITLE
Add conan file

### PR DESCRIPTION
**Pull request by sairoopsomaraju**

### CODE REVIEW

File: conanfile.py [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+    options = {
+        "shared": [True, False]
+    }
+    default_options = {
+        "shared": False
+    }
+
+    def build(self):
+        if self.settings.os.distribution == "ubuntu":
+            self.run("apt-get update")
```
Comment (katikalathejakiran) [inline conanfile.py:23]: why is this required??

File: conanfile.py [hunk 2]
```diff
@@ -0,0 +1,48 @@
+from conans import ConanFile, CMake, tools
+
+class cmockalibConan(ConanFile):
+    name = "kinesiscproducer"
+    version = "0.1.0"
```
Comment (katikalathejakiran) [inline conanfile.py]: this should be 1.0.0

**Review by katikalathejakiran — CHANGES_REQUESTED** [review]

File: conanfile.py [hunk 3]
```diff
@@ -0,0 +1,48 @@
+from conans import ConanFile, CMake, tools
```
Comment (katikalathejakiran) [inline conanfile.py:1]: why do we a have elear-develop??

**Review by katikalathejakiran — CHANGES_REQUESTED** [review]

File: conanfile.py (hunk 1 above)
Comment (sairoopsomaraju) [reply conanfile.py:23]: Kinesis needs git to be preinstalled

File: conanfile.py (hunk 2 above)
Comment (sairoopsomaraju) [reply conanfile.py]: ok

File: conanfile.py (hunk 3 above)
Comment (sairoopsomaraju) [reply conanfile.py:1]: that's the naming convention for develop branches which are for open source repos

**Review by katikalathejakiran — APPROVED** [review]
